test/Utils/UnitSpec.scala

```diff
@@ -55,7 +55,7 @@ trait UnitSpec
     with GuiceOneAppPerSuite
 {
 
-  val dateTime: LocalDateTime = LocalDateTime.now()
+  val dateTime: LocalDateTime = LocalDateTime.now().truncatedTo(java.time.temporal.ChronoUnit.MILLIS)
```

Comment:
Happy to approve, but please just add a comment above this explaining why we need to do it.  See my example in Slack... thanks!